app/scripts/styles/theme.js

```diff
@@ -10,7 +10,8 @@ export default function themeOverrides() {
       infographicA: '#fcab10',
       infographicB: '#f4442e',
       infographicC: '#b62b6e',
-      infographicD: '#2ca58d'
+      infographicD: '#2ca58d',
+      infographicE: '#2276ac'
```

Comment:
@ricardoduplos Can you provide more infographic colors? Or should we use a generator function?